ユーザ登録画面のデザインの作成

koron56 opened the pull request:
新規登録の画面のデザインを追加した
白色ベースのホーム画像の写真を追加した

koron56 committed:
add signup_form.php, 01_0.svg
新規登録の画面のデザインを追加した
白色ベースのホーム画像の写真を追加した

reiya97 reviewed (changes requested):
signup_form.phpの23行目のcssの読み込みURLが間違えてます
login_form/main.css → signup_form/main.css

koron56 committed:
ユーザ登録画面の変更

signup_form.phpの23行目のcssの読み込みURLの変更
login_form/main.css → signup_form/main.css

スクロールバーの追加

reiya97 reviewed (approved):
確認しました．オッケー
